.github/workflows/ci.yml

```
@@ -58,6 +58,66 @@ jobs:
           env-vars-for-codebuild: JAVA_ENV_VERSION
         env:
           JAVA_ENV_VERSION: ${{ matrix.platform.distribution }}${{ matrix.version }}
+  vectorTestsMasterKey:
```

I thought this was just loading Keys, but it is testing the legacy Master Key Provider interfaces against the test vectors, correct?
```suggestion
  vectorTestsMasterKeyProvider:
```